bsp/worker/src/mill/bsp/worker/MillBuildServer.scala

```diff
@@ -234,12 +234,14 @@ private class MillBuildServer(
         case module: MillBuildRootModule =>
```

Comment:
Not sure why this warrants a specific case, considering it extends JavaModule anyway

Reply:
Could be some left over from a time, where `MillBuildRootModule` was either not a `JavaModule` or needed different treatment. If both cases are identical now, we can merge them.